enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -170,7 +172,19 @@
     // LDAP authorization settings
     //-----------------------------------------------------
 
-    @Description( "Perform LDAP search for authorization info using a system account." )
+    @Description( "Perform LDAP search for authorization info using a system account or the user's own account.\n\n" +
```

Comment:
Or:
"Perform LDAP search for authorization info using a system account instead of the user's own account.\n\n"